haas/cli.py

```diff
@@ -180,17 +186,22 @@ def setup_http_client():
     This may be extended with other backends in the future.
     """
```

Comment:
The docstring for this function will need to be updated to reflect the changes this patch makes. I would wait until after you've sorted out the auth stuff I mentioned though, as that's likely to involve further changes to exactly what this does.